tests/backup/backup_portworx_test.go

```diff
@@ -582,3 +584,279 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", func() {
+					pods, err := core.Instance().GetPods(namespace, labelSelectors)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the pod list"))
+					srcClusterConfigPath, err := GetSourceClusterConfigPath()
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting kubeconfig path for source cluster %v", srcClusterConfigPath))
+					for _, pod := range pods.Items {
+						afterSize, err := getSizeOfMountPoint(pod.GetName(), namespace, srcClusterConfigPath)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the mount size %v from pod %v", afterSize, pod.GetName()))
+						podListAfterSizeMap[pod.Name] = afterSize
+					}
+					for _, pod := range pods.Items {
+						dash.VerifyFatal(podListAfterSizeMap[pod.Name] > podListBeforeSizeMap[pod.Name], true, fmt.Sprintf("Verifying volume size has increased for pod %s", pod.Name))
+					}
+				})
+				Step("Verifying backup success after initializing volume resize", func() {
+					log.InfoD("Verifying backup success after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					firstScheduleBackupName, err = GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+					err = backupSuccessCheck(firstScheduleBackupName, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the success of the first schedule backup named [%s]", firstScheduleBackupName))
+					allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching all schedule backups %v", allScheduleBackupNames))
+					currentScheduleBackupCount := len(allScheduleBackupNames)
+					log.InfoD("Current number of schedule backups is [%v]", currentScheduleBackupCount)
+					nextScheduleBackupOrdinal := currentScheduleBackupCount + 1
+					log.InfoD("Ordinal of the next schedule backup is [%v]", nextScheduleBackupOrdinal)
+					log.InfoD("Waiting for 15 minutes for the next schedule backup to be triggered")
+					time.Sleep(15 * time.Minute)
+					checkOrdinalScheduleBackupCreation := func() (interface{}, bool, error) {
+						ordinalScheduleBackupName, err := GetOrdinalScheduleBackupName(ctx, scheduleName, nextScheduleBackupOrdinal, orgID)
+						if err != nil {
+							return "", true, err
+						}
+						return ordinalScheduleBackupName, false, nil
+					}
+					nextScheduleBackupName, err = task.DoRetryWithTimeout(checkOrdinalScheduleBackupCreation, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching next schedule backup name of ordinal [%v] of schedule named [%s]", nextScheduleBackupOrdinal, scheduleName))
+					log.InfoD("Next schedule backup name [%s]", nextScheduleBackupName.(string))
```

Comment:
Don't use `.(string)` repeatedly. Convert the value to a string and use it everywhere.